src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -491,4 +490,44 @@ extern void cprLookupByAttribute(EntityId&                          en,
                                  std::string*                       perEntPa,
                                  std::string*                       perAttrPa);
 
+
+/* ****************************************************************************
+*
+* basePart, idPart -
+*
+* Helper functions for entitysQuery to split the attribute name string into part,
+* e.g. "A1__ID1" into "A1" and "ID1"
+*/
+inline std::string basePart(std::string name)
+{
+  /* Search for "__" */
+  std::size_t pos = name.find("__");
+  if (pos == std::string::npos)
+  {
+    /* If not found, return just 'name' */
+    return name;
+  }
+  else
```

Comment:
4 lines => 1 line

In general, if an 'if' ends in return, else is not necessary ...